.github/workflows/build.yml

```diff
@@ -78,7 +78,6 @@ jobs:
       CCACHE_MAXSIZE: 0
       CCACHE_NOCOMPRESS: 1
       MAKEFLAGS: -j3
-      DEVELOPER_DIR: /Applications/Xcode_11.7.app/Contents/Developer
```

Comment:
This was specified in order to use the oldest Xcode version available, to allow the application to run on older macOS versions. For example, the default Xcode version on the macOS 12 image is 14.2, which targets macOS 13.1. We can specify a minimum of 13.1, which targets macOS 12.0 instead.

Reply:
Gotcha. Maybe it'd be good to document this, I couldn't find any explanation in the code or commit history. Feel free to re-add the proposed `DEVELOPER_DIR` value and merge if you get to it before I do. I'm signing off for today.

Reply:
> Xcode version [...] 14.2 [...] targets macOS 13.1. We can specify a minimum of 13.1, which targets macOS 12.0 instead.

I'd like to familiarize myself with this, can you point me to a source or documentation? All I can find is this [table from Apple](https://developer.apple.com/support/xcode), which lists macOS 10.13 as the minimum deployment target for Xcode 14.2.

Reply:
Anyway, added it back in with `13.1` in the interest of minimal change - guess it won't hurt. Merging.